cdk.json

```
@@ -36,6 +36,7 @@
       "aws-cn"
     ],
     "dev": {
+      "DreamkastApiBaseUrl" : "https://dreamkast-dk-1621.dev.cloudnativedays.jp",
```

このreviewappsはすでに消えているので、変えたほうがよさそうです。

```suggestion
      "DreamkastApiBaseUrl" : "https://staging.dev.cloudnativedays.jp",
```

---

コメントありがとうございます。修正しました！